operator/Makefile

```diff
@@ -229,14 +229,14 @@ vet: ## Run go vet against code.
 	go vet ./...
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.25
+ENVTEST_K8S_VERSION = 1.27
 
 .PHONY: test
-test: manifests generate fmt vet  ## Run go unit tests
-	go test $(shell go list ./... | grep -v '^github.com/stackrox/rox/operator/tests') -coverprofile cover.out
+test: manifests generate fmt vet envtest  ## Run go unit tests
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(shell go list ./... | grep -v '^github.com/stackrox/rox/operator/tests') -coverprofile cover.out
```

Comment:
Nit: Hm, I thought we have the separation between `test` and `test-integration` precisely because the former don't require envtest (and are therefore somewhat faster to run).
Do both need envtest now? 🤔

Reply:
Indeed we don't need them, reverted the change